sdk/servicebus/service-bus/src/serviceBusMessage.ts

```diff
@@ -438,7 +438,7 @@ export interface ServiceBusReceivedMessage extends ServiceBusMessage {
    * @property {AmqpMessage} _amqpMessage The underlying raw amqp message.
    * @readonly
    */
-  readonly _amqpAnnotatedMessage: AmqpAnnotatedMessage;
+  readonly _amqpAnnotatedMessage?: AmqpAnnotatedMessage;
```

Comment:
Making this optional will not be an accurate representation of a received message. I would recommend reverting the commit https://github.com/Azure/azure-sdk-for-js/pull/12091/commits/a7455499fe3de69e8ba451d7e9c2b7bdc5c5c133 and revisiting the refactoring later.

Reply:
Done